tests/test_data/americastestkitchen.com/americastestkitchen.json

```diff
@@ -0,0 +1,49 @@
+{
+  "author": "America's Test Kitchen",
+  "canonical_url": "https://www.americastestkitchen.com/recipes/13980-instant-pot-mussels-with-fennel-and-leeks",
+  "category": "Main Courses, Weeknight",
+  "description": "Your Instant Pot is actually a great place to cook seafood.",
+  "host": "americastestkitchen.com",
+  "image": "https://res.cloudinary.com/hksqkdlah/image/upload/ar_1:1,c_fill,dpr_2.0,f_auto,fl_lossy.progressive.strip_profile,g_faces:auto,q_auto:low,w_150/SFS_Mussels-with-Fennel-and-Leeks_6_5_xjdlfl",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 tablespoon, extra-virgin olive oil, plus extra for drizzling",
+        "1 , fennel bulb, 1 tablespoon fronds minced, stalks discarded, bulb halved, cored, and sliced thin",
+        "1 , leek, ends trimmed, leek halved lengthwise, sliced 1 inch thick, and washed thoroughly",
+        "4 , garlic cloves, minced",
+        "3 sprigs, fresh thyme",
+        "1/4 teaspoon, red pepper flakes",
+        "1/2 cup, dry white wine",
+        "3 pounds, mussels, scrubbed and debearded"
+      ],
+      "purpose": null
+    }
+  ],
+  "ingredients": [
+    "1 tablespoon, extra-virgin olive oil, plus extra for drizzling",
+    "1 , fennel bulb, 1 tablespoon fronds minced, stalks discarded, bulb halved, cored, and sliced thin",
+    "1 , leek, ends trimmed, leek halved lengthwise, sliced 1 inch thick, and washed thoroughly",
+    "4 , garlic cloves, minced",
```

Comment:
This output doesn't seem ideal - I guess the `schema.org` data on the page has some quirks compared to the results constructed from the `__NEXT_DATA__` datastructures.

I'd _probably_ recommend a regular expression here to match on something like: start-of-string, a series of digits, space-comma-space.

An alternative might be to try to construct these from the `__NEXT_DATA__` contents, but I think that's probably too complicated (unless there are many other formatting edge cases like this).

Reply:
How embarrassing.  I didn't look terribly close at the output from that and just assumed it was formatted sanely.

The other data format (from `_get_additional_details()`) is already in ingredient groups.  I'll reimplement using that and add ingredient_groups.

Reply:
No problem!  Yep, re-using that code sounds good to me :+1: